.env.example

```
@@ -19,21 +19,52 @@ SSLCOMMERZ_VALIDATOR_ENDPOINT=
 SSLCOMMERZ_STORE_ID=
 SSLCOMMERZ_STORE_PASSWORD=
 
-# Provider configuration (Ollama + Groq)
+# Provider configuration
 PROVIDER_TIMEOUT_MS=4000
 PROVIDER_MAX_RETRIES=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PROVIDER_MAX_RETRIES key should go before the PROVIDER_TIMEOUT_MS key

(UnorderedKey)